HelpSource/Classes/GridLines.schelp

```diff
@@ -110,6 +110,9 @@ If numTicks is nil: used to guess the ideal numTicks based on the graph size.
 argument:: numTicks
 Explicit number of ticks you would like to see on the graph.
 
+argument:: tickSpacing
+Minimal spacing to maintain between ticks. Only used if code::numTicks:: is code::nil::. For an exponential grid it is used for the lowest portion of each decade.
```

Comment:
I think it's good that `tickSpacing` takes precedence over `numTicks`, i.e. there can be fewer ticks than specified by `numTicks` in the case that the plot is small and the ticks spacing is below the `tickSpacing` threshold. That way, `tickSpacing` enforces a maximum density of grid lines.

But I don't think it should be ignored `numTicks` is specified (not `nil`). A user can effectively disable the `tickSpacing` overriding `numTicks` by just setting `tickSpacing = 0` (which would be good to note in the docs).

Reply:
> But I don't think it should be ignored numTicks is specified (not nil)

I'm not sure if I have a good idea how to change the implementation to accommodate this request. Do you still want me to add this to this PR?

Reply:
No, let's leave this to a future PR, I'll try to get that going in the coming days, accounting for the discussion here.

Thanks!